source/test/F0.Minesweeper.Logic.Tests/LocationTest.cs

```diff
@@ -27,7 +25,7 @@ public void NewLocation_YParameterEqualsY()
 		[Fact]
 		public void NewLocation_MaxIntParameterOnX()
 		{
-			uint testValue = int.MaxValue;
+			uint testValue = Int32.MaxValue;
```

Comment:
Is there an analyzer/editorconfig setting that we can enable, to get a warning if we use the keywords, instead of the full name for this?

Reply:
Yes: see #15
We may tackle this after https://github.com/Flash0ver/F0.Minesweeper/milestone/2